src/Magento/FunctionalTestingFramework/DataGenerator/Handlers/CredentialStore.php

```diff
@@ -57,16 +63,28 @@ private function __construct()
         $csToken = getenv('CREDENTIAL_VAULT_TOKEN');
         if ($csBaseUrl !== false && $csToken !== false) {
             try {
-                $this->credVault = new VaultStorage(rtrim($csBaseUrl, '/'), $csToken);
+                $this->credStorage[self::ARRAY_KEY_FOR_VAULT] = new VaultStorage(
+                    rtrim($csBaseUrl, '/'),
+                    $csToken
+                );
             } catch (TestFrameworkException $e) {
             }
         }
 
         // Initialize file storage
         try {
-            $this->credFile = new FileStorage();
+            $this->credStorage[self::ARRAY_KEY_FOR_FILE]  = new FileStorage();
         } catch (TestFrameworkException $e) {
```

Comment:
I think the exceptions are already logged in FileStorage and VaultStorage.

Reply:
Sure, but human readable statements will make the log more useful. Instead of just seeing a failure message, you see that the credential store is *not* using a specific method.